src/main/java/us/spaceclouds42/disableportals/mixin/EndGatewayBlockEntityMixin.java

```diff
@@ -17,7 +20,7 @@ abstract class EndGatewayBlockEntityMixin {
             ),
             cancellable = true
     )
-    private void disableEndGateway(Entity entity, CallbackInfo ci) {
+    private static void disableEndGateway(World world, BlockPos pos, BlockState state, Entity entity, EndGatewayBlockEntity blockEntity, CallbackInfo ci) {
```

Comment:
...does that work? That doesn't look like it would work.

Reply:
tryTeleportingEntity is now static